ext/dom/php_dom.stub.php

```diff
@@ -303,7 +303,7 @@ public function loadXML(string $source, int $options = 0) {}
     public function normalizeDocument() {}
 
     /** @return bool */
-    public function registerNodeClass(string $baseclass, ?string $extendedclass) {}
+    public function registerNodeClass(string $basecClass, ?string $extendedClass) {}
```

Comment:
Is this `$basecClass` correct or should it be `$baseClass`?

Reply:
Oh good catch, thanks